### PR TITLE
docs(stats): Captain's log update

### PR DESCRIPTION
**Pull request by dnilasor**
Added more specific analysis to text accompanying graphs to highlight key takeaways from each visualization. Added to and simplified introduction.

### CODE REVIEW

**Comment by dnilasor** [comment]
also fixed all the subheadings per style guide

File: community/captains-log/index.md [hunk 1]
```diff
@@ -4,17 +4,16 @@ title:  "Captain's Log: The State of Spinnaker"
 sidebar:
   nav: community
 ---
+_Welcome to the first report on the state of Spinnaker. This Captain's Log contains regularly updated project contribution and code repository metrics, to visualize the story of the Spinnaker's growth across its ecosystem._
```
Comment (dorbin) [inline community/captains-log/index.md]: "...first report..."

I think this is ok, but keep in mind you (or we)'ll have to either increment it each time or just remove it when posting the second report.

File: community/captains-log/index.md [hunk 2]
```diff
@@ -26,52 +25,54 @@ Our community began as a partnership between Netflix & Google, eager to share th
...
   include
   figure
   image_path="./company_dev_count.png"
 %}
 
-### Myriad Pull Request Contributors
+### Myriad pull request contributors
 
-Unique authors contribute pull requests to Spinnaker repositories each week, building the project collaboratively over time.
+Unique authors contribute pull requests to Spinnaker repositories each week, building the project collaboratively over time. 314 developers submitted PRs in 2019, compared to 282 in 2018, and 194 in 2017:
```
Comment (dorbin) [inline community/captains-log/index.md]: s/Unique/Many/

In this context, it sounds like the authors are...unusual.

Comment (dorbin) [inline community/captains-log/index.md]:
```suggestion
Unique authors contribute pull requests to Spinnaker repositories each week, building the project collaboratively over time. In 2019, 314 developers submitted PRs, compared to 282 in 2018, and 194 in 2017:
```

Comment (dorbin) [reply community/captains-log/index.md]: Suggesting that change just fto avoid starting a sentence with a numeral.

**Review by dorbin — APPROVED** [review]
Thanks, Rosalind. A few minor suggestions, but approving now.

File: community/captains-log/index.md (hunk 2 above)
Comment (dnilasor) [reply community/captains-log/index.md]: good call

File: community/captains-log/index.md (hunk 1 above)
Comment (dnilasor) [reply community/captains-log/index.md]: I'd rather not introduce this, good catch

**Pull request merged**